azuracast/liquidsoap_custom.liq

```diff
@@ -413,76 +413,76 @@ radio = switch(track_sensitive=false, transitions=[
-    ({3w and 18h-18h58m}, once(playlist_boombap)),
-    ({3w and 19h-19h58m}, once(playlist_eastern_daze)),
-    ({3w and 20h-20h58m}, once(playlist_1800venus)),
-    ({3w and 21h-21h58m}, once(playlist_linear_systems_with_gestalt)),
-    ({3w and 12h15m-21h30m}, off_air_ambient_mix),                     #ambient: Tudtad - Gestalt
-    ({4w and 10h-11h58m}, once(playlist_ritka_csut)),                  #Thursday
-    ({4w and 14h-14h58m}, once(playlist_fkse)),
-    ({4w and 15h-16h58m}, once(playlist_lazy_calm_raga)),
-    ({4w and 17h-17h58m}, once(playlist_dalmata_gergo_show)),
-    ({4w and 18h-18h58m}, once(playlist_keygen)),
-    ({4w and 19h-20h58m}, once(playlist_schmerz)),
-    ({4w and 21h-22h28m}, once(playlist_gyogyfurdo)),
-    ({4w and 12h30m-21h30m}, off_air_ambient_mix),                        #ambient: Ritka csut - Gyogyf
-    ({5w and 10h-11h58m}, once(playlist_ritka_pentek)),                #Friday
-    ({5w and 12h00m-12h28m}, once(playlist_melania)),                     
-    ({5w and 12h30m-13h58m}, once(playlist_szusivacsi)),
-    ({5w and 14h-14h58m}, once(playlist_randomcsunyalanyok)),           
-    ({5w and 15h-15h58m}, once(playlist_dead_hound)),
-    ({5w and 16h-16h58m}, once(playlist_graveyard_slot)),
-    ({5w and 17h-17h58m}, once(playlist_brainfuq)),
-    ({5w and 18h-18h58m}, once(playlist_moneyka)),
-    ({5w and 19h-20h58m}, once(playlist_arcadeok_alatt)),       
-    ({5w and 12h15m-21h30m}, off_air_ambient_mix),                        #ambient: Ritka pentek - Rev
-    ({6w and 9h30m-11h28m}, once(playlist_merites)),                  #Saturday
-    ({6w and 11h30m-12h28m}, once(playlist_jambikus_nesz)),     
-    ({6w and 12h30m-14h58m}, once(playlist_torso)),
-    ({6w and 15h-15h58m}, once(playlist_footwork_jimbob)),
-    ({6w and 16h-17h58m}, once(playlist_rnr666)),
-    ({6w and 18h-19h58m}, once(playlist_mood_sequence)),
-    ({6w and 21h-22h28m}, once(playlist_d23)),
-    ({6w10h-6w23h}, off_air_ambient_mix),                     #ambient: Merites - Gestalt
-    ({7w and 8h30m-10h28m}, once(playlist_lohuma)),                    #Sunday
-    ({7w and 10h30m-11h58m}, once(playlist_bambusz)),
-    ({7w and 12h-14h58m}, once(playlist_donald_kacsa_klub)),
-    ({7w and 15h-15h58m}, once(playlist_geigercounterculture)),
-    ({7w and 16h-16h58m}, once(playlist_hetedik_tipusu_talalkozas)),
-    ({7w and 17h-17h58m}, once(playlist_hatterzaj)),
-    ({7w and 18h-18h58m}, once(playlist_csuszka)),
-    ({7w and 19h-19h58m}, once(playlist_sub_burek)),
-    ({7w and 20h-20h58m}, once(playlist_zeneszen)),
-    ({7w and 9h-20h30m}, off_air_ambient_mix),                        #ambient: Lohuma - Zeneszen
+    ({1w and 9h-9h58m}, once(playlist_szedd_a_labad)),             # Monday
+    ({1w and 10h11m-11h9m}, once(playlist_transverszia)),             # orig requested time 11:11 am 12:12 pm
+    ({1w and 12h-12h58m}, once(playlist_a_who_say)),
+    ({1w and 13h-14h28m}, once(playlist_korunk)),    
```

Comment:
Thanks for paying attention to these changes on the other branch!

Reply:
This looked like the easiest way to go. I searched for [an article](https://hu.euronews.com/my-europe/2022/10/28/mikor-szunik-meg-az-eu-ban-az-oraatallitas-az-ido-lesz-a-megmondhatoja) about the EU topic, it says that the decision was delayed because of covid and the war now, and probably it will be introduced around 2025.

Reply:
> Thanks for paying attention to these changes on the other branch!

If you mean Korunk & Önkutató, we merged that branch into master before Endre did the clock changes.